packages/main/src/Dialog.ts

```diff
@@ -261,6 +261,8 @@ class Dialog extends Popup {
 	 *
 	 * @param {boolean} preventInitialFocus Prevents applying the focus inside the popup
 	 * @async
+	 * @method
```

Comment:
the rest are addressed with https://github.com/SAP/ui5-webcomponents/pull/6304, but nevertheless thank you